dotnet/src/Connectors/Connectors.Memory.Weaviate/Diagnostics/NullableAttributes.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// This was copied from https://github.com/dotnet/runtime/blob/39b9607807f29e48cae4652cd74735182b31182e/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs
+// and updated to have the scope of the attributes be internal.
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
```

Comment:
Could also reference the `  <Import Project="$(RepoRoot)/dotnet/src/InternalUtilities/InternalUtilities.props" />` to not have to duplicate this, I think.